application/controllers/admin/globalsettings.php

```diff
@@ -143,6 +143,12 @@ private function displaySettings()
         $data['sideMenuBehaviour'] = getGlobalSetting('sideMenuBehaviour');
         $data['aListOfThemeObjects'] = AdminTheme::getAdminThemeList();
 
+        // List of available OAuth plugins
```

Comment:
Question : can not be more global ? I explain after  1/2 

```
$event = new PluginEvent('listSMTPOAuthPlugins', $this);
$event->get('mailerplugins', []); //just to be clear
Yii::app()->getPluginManager()->dispatchEvent($event);
$mailerplugins = $event->get('mailerplugins');
```

(we still have POP3 not included , and we can use more easily other phpMailer extended class)

Reply:
I understand what you say. We want to analyze it a bit.
Do you have a concrete use case?

Reply:
PS : must be 

1. listMailPlugins (as an array)
2. afterSelectMailPlugin
3. MailerConstruct

Something near export : exportPlugins is cool things to do.